Frameworks/CoreFoundation/String.subproj/CFString.c

```diff
@@ -1192,6 +1192,11 @@ CFHashCode __CFStringHash(CFTypeRef cf) {
     }
 }
 
+// WINOBJC: _CFStringHash is for Foundation's benefit. It should not be consumed by anyone else.
+CF_PRIVATE CF_EXPORT CFHashCode _CFStringHash(CFStringRef str) {
```

Comment:
ForFoundationOnly.h is a file from the swift open-source, we probably ought to mess with it as little as possible. CFFoundationInternal.h has been where we've been putting functions of this nature.